dogowner/models.py

```diff
@@ -0,0 +1,78 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Gender(models.TextChoices):
+    Male = 'M', 'Male'
+    Female = 'F', 'Female'
+    Unknown = 'UN', 'Unknown'
+
+
+class Dogowner(models.Model):
+
+    user = models.ForeignKey(User,
+                             on_delete=models.CASCADE,
+                             blank=True,
+                             null=True
+                             )
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    phone_number = models.CharField(max_length=20)
```

Comment:
we can assume it's max length is 10 digits.
it should probably also be integers.
we can ignore international numbers for now IMO (but even then, 20 Is too long)

Reply:
Fixed